src/verdant/panel/events/event.tsx

```diff
@@ -35,51 +31,28 @@ class NotebookEvent extends React.Component<NotebookEvent_Props> {
           />
         </div>
         <div className={`${COL} map`}>
-          <div className={EVENT_MAP}>
-            <div className={EVENT_NOTEBOOK}>{`# ${
-              this.props.events.notebook + 1
-            }`}</div>
-            <div className={EVENT_NOTEBOOK}>{this.showMap()}</div>
+          <div className={EVENT_INDEX_LABEL}>
+            {`# ${this.props.events.notebook + 1}`}
+          </div>
+          <div className={EVENT_MAP_LABEL}>
+          <NotebookEventMap
+            checkpoints={this.props.events.events}
+            history={this.props.history}
```

Comment:
NotebookEventMap should get history directly from the redux store. If history is not used elsewhere here, just remove it as a prop for Event